modules/custom-status/custom-status.php

```diff
@@ -539,11 +542,13 @@ public function update_custom_status( $status_id, $args = [] ) {
 			}
 		}
 		// We're encoding metadata that isn't supported by default in the term's description field
-		$args_to_encode                = [];
-		$args_to_encode['description'] = ( isset( $args['description'] ) ) ? $args['description'] : $old_status->description;
-		$args_to_encode['position']    = ( isset( $args['position'] ) ) ? $args['position'] : $old_status->position;
-		$encoded_description           = TaxonomyUtilities::get_encoded_description( $args_to_encode );
-		$args['description']           = $encoded_description;
+		$args_to_encode                       = [];
+		$args_to_encode['description']        = $args['description'] ?? $old_status->description;
+		$args_to_encode['position']           = $args['position'] ?? $old_status->position;
+		$args_to_encode['is_review_required'] = $args['is_review_required'] ?? $old_status->is_review_required ?? false;
```

Comment:
I think this [double null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op) is pretty nifty.